### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Edreih**
Add Anchor (Money, Budgeting and Management) and Solo (Project Management)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -776,6 +777,7 @@ See also [Ticketing](#ticketing), [Task management/To-do lists](#task-management
   * [OpenProject](https://www.openproject.org) - Project collaboration.
   * [Phabricator](http://phabricator.org/) - Collection of web applications that help build better software. ([Demo](https://secure.phabricator.com/), [Source Code](https://github.com/phacility/phabricator)) `Apache` `PHP`
   * [Redmine](http://www.redmine.org/) - Redmine is a flexible project management web application. `Ruby` `GPLv2`
+  * [Solo](http://www.getsoloapp.com/) - Solo is a free project management app created for freelancers. Create contacts, manage tasks, upload files, track project progress, and keep notes. ([Demo](http://www.getsoloapp.com/demo/)), `PHP`
```
Comment (nodiscc) [inline README.md]: 1. There is no source code available
2. > This software can not be redistributed in any form.

Please move this to non-free.md

There's also an unneeded `,` at the end.

File: README.md [hunk 2]
```diff
@@ -615,6 +615,7 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
 
 ## Money, Budgeting and Management
 
+  * [Anchor](http://theanchorapp.com/) - Anchor is invoicing software that you host on your own server and is free to use forever ([Demo](http://theanchorapp.com/demo/admin-login)), `PHP`
```
Comment (nodiscc) [inline README.md]: 1. There's no source code
2. > This software can not be redistributed in any form.

Please move this to non-free.md

There's also an unneeded , at the end.

File: README.md [hunk 3]
```diff
@@ -1011,6 +1013,7 @@ The list of contributors can be found below
     3	Chris McCormick <chris@mccormick.cx>
     3	Conor O'Callaghan <brioscaibriste@users.noreply.github.com>
     3	Cédric Krier <cedk@users.noreply.github.com>
+    3 Edreih Aldana <admin@ealdana.com>
```
Comment (nodiscc) [inline README.md]: No need to do this, it's updated periodically using `git shortlog -sne`

**Comment by nodiscc** [comment]
@Edreih unfortunately these are non-Free Software applications. Please fix as requested in the comments above.

**Comment by Kickball** [comment]
#588 resolves the issues @nodiscc has identified.

**Pull request closed without merging**